src/AppBundle/Subscriber/BugLabelNewIssueSubscriber.php

```diff
@@ -36,15 +36,15 @@ public function onIssues(GitHubEvent $event)
         }
 
         $responseData = array('issue' => $issueNumber = $data['issue']['number']);
-        // Ignore non-bugs or issue which already has a status
+        // Ignore non-bugs or issue which already has a statu
         if ('bug' !== strtolower($data['label']['name']) || null !== $this->statusApi->getIssueStatus($issueNumber, $repository)) {
             $responseData['status_change'] = null;
             $event->setResponseData($responseData);
 
             return;
         }
 
-        $newStatus = Status::NEEDS_REVIEW;
+        $newStatus = Status::UNCONFIRMED;
```

Comment:
Yeah I thought about this alternative :)
But as pointed in the ticket, IMHO issues don't need a review in general, more opinions, confirmations and maybe support. At least for Bugs, we are sure they just need confirmation, so maybe [this one](https://github.com/carsonbot/carsonbot/pull/45#issuecomment-243268636) can be a good compromise. more expressive imo, wdyt?